lib/keys.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Creates keys for `private` properties on exposed objects to minimize interactions with other codebases.
+ * The key will be a Symbol if the host supports it; otherwise a prefixed string.
+ */
+var create = 
+  typeof Symbol !== 'undefined' ? 
+  function(name) {
+    return Symbol(name);
+  } 
+  :
+  function(name) {
+    return '_zone$' + name;
+  };
+
+var commonKeys = {
+  originalAddEventListener: create('originalAddEventListener'),
```

Comment:
1) make key names UPPER_CASE ?

2) IMO `original` can be dropped, it has no added value.